clang_format.py

```diff
@@ -108,7 +108,7 @@ def main():
     git_cf_path = os.path.join(this_dir, "git-clang-format")
     print('Formatting ' + ' '.join(sys.argv[1:]))
     subprocess.run(
-        [git_cf_path, "-f", "--binary", str(clang_format_path()), "--"] + sys.argv[1:]
+        [python, git_cf_path, "-f", "--binary", str(clang_format_path()), "--"] + sys.argv[1:]
```

Comment:
Where does this variable `python` come from?  When I run this, I get

```
Traceback (most recent call last):
  File "/var/folders/y8/8280zwdj7g76b0gtfjmkv0br0000gp/T/tmpaqn6igb4/repoel_6ps6e/clang_format.py", line 116, in <module>
    main()
  File "/var/folders/y8/8280zwdj7g76b0gtfjmkv0br0000gp/T/tmpaqn6igb4/repoel_6ps6e/clang_format.py", line 111, in main
    [python, git_cf_path, "-f", "--binary", str(clang_format_path()), "--"] + sys.argv[1:]
NameError: name 'python' is not defined
```

Reply:
It isn't a variable, it is a command and apparently when I copied it to here from my working dir I forgot to include the quotes around it.

Reply:
OK!  I think the right thing is `sys.executable`; I'll merge a PR with that change and with the change for the temp filesystem issue.

Thank you again for the PR, I feel awful that I didn't see this for so long.